src/utils/pr-text.ts

```diff
@@ -0,0 +1,55 @@
+import { ROLL_TARGETS, RollTarget } from '../constants';
+
+interface PRTextDetails {
+  previousVersion: string;
+  newVersion: string;
+  branchName: string;
+}
+
+export function getPRText(rollTarget: RollTarget, details: PRTextDetails) {
+  switch (rollTarget.name) {
+    case ROLL_TARGETS.NODE.name:
+      return getNodePRText(details);
+    case ROLL_TARGETS.CHROMIUM.name:
+      return getChromiumPRText(details);
+  }
```

Comment:
Can we `throw` here in the default case to ensure that we get a clear mis-use error if something crazy happens